apps/explorer/src/hooks/useGetTransactionBlocks.ts

```diff
@@ -32,7 +32,10 @@ export function useGetTransactionBlocksForAddress(
         {
             getNextPageParam: (lastPage) =>
                 lastPage?.hasNextPage ? lastPage.nextCursor : false,
-            enabled: !!address,
+            staleTime: Infinity,
+            cacheTime: 24 * 60 * 60 * 1000,
+            retry: false,
+            keepPreviousData: true,
```

Comment:
Does this apply for `useInfiniteQuery` too?

Reply:
Yep, works the same as the `useQuery` one